content/vendor/guides/airgapped-cluster.md

```diff
@@ -107,35 +107,33 @@ ssh -N -L 8800:${AIRGAP_PRIVATE_IP}:8800 ${JUMPBOX_PUBLIC_IP}
 
 * * *
 
-## Existing Kubernetes Cluster
+## Icon in Base64
 
-Installing into an airgapped existing kubernetes cluster is a similar experience to an [online existing cluster](/vendor/guides/existing-cluster) install. There is no kURL bundle needed for existing installs because the expectation is for the systems administrator to install and maintain the Kubernetes components of their EKS, GKE, AKS, OpenShift, etc. clusters.
+In an Airgap environment, if the `icon` uses a URL, it would not be able to display because the image is fetched at the time the page is rendered. In Airgap you need to convert the `png` file into `base64` to be used for the image.
 
-### Install via kubectl
+### Converting Icon to Base64
 
-* Once you have your context configured to the airgapped kubernetes cluster, run `kubectl kots install` to install the open source Admin Console components.
+Using `curl` we fetch the image, then use `openssl base64` to encode the image into Base64 and then `pbcopy` to have it copied to your local clipboard.
 
 ```shell
-$ kubectl kots install <your-app-name-and-channel>
+curl -LSs https://sentry-brand.storage.googleapis.com/sentry-glyph-black.png | openssl base64 | pbcopy
```

Comment:
amazing

Reply:
Maybe we shouldn't use `pbcopy` here, since that's only available on macOS?

Reply:
@austinchambers I was thinking about that, what would be the best way to approach this?